assets/css/main.css

```diff
@@ -57,3 +71,27 @@ footer {
   margin-bottom: -20px;
   padding: 20px 0 20px 0;
 }
+
+.SocialIcons {
```

Comment:
CSS classes name should be kept consistent with casing.

Instead of `SocialIcons`, please use `social-icons`